config/locales/en.yml

```diff
@@ -32,6 +32,11 @@ en:
     private_topics:
       errors:
         user_ids_length: Please specify at least one other user.
+    search:
+      form:
+        btn_submit: :thredded.search.form.label
```

Comment:
btn is a common abbreviation of button, popularized by Bootstrap. Example: [Discourse](https://github.com/discourse/discourse/blob/master/config/locales/client.en.yml#L479).